README.md

```diff
@@ -25,7 +25,7 @@
 
 # 📖 [MEME-GALAXY] <a name="about-project"></a>
 
-**Meme Galaxy** is a single-page app built using React JS and Vite whose aim is to display different memes with differenct categories accodrding to data fetched from <b>imgflip API.</b> Further information is available in the <a href="https://drive.google.com/file/d/1eZNRTECZUpxfykamc0erwobgSFsP5tY5/view?usp=sharing">video presentation</a>
+**Meme Galaxy** is a single-page app built using React JS and Vite whose aim is to display different memes with differenct categories accodrding to data fetched from <b>imgflip API.</b> For more information visit the <a href="https://drive.google.com/file/d/1eZNRTECZUpxfykamc0erwobgSFsP5tY5/view?usp=sharing">video presentation</a>
```

Comment:
**I like the project you made!**

But in your design:

- [ ] 1. I cannot see the icons
- [ ] 2. The title is not aligned as expected
- [ ] 3. Wireframe does not look as expected
- [ ] 4. The top bar has some differences


Your image            |  Original image
:-------------------------:|:-------------------------:
<img width="1440" alt="Screenshot 2023-01-03 at 18 29 21" src="https://github.com/MarwanDev/Meme-Galaxy/assets/69011963/7358621c-5931-42b0-aa0a-095148695e25"> | <img src="https://github.com/MarwanDev/Meme-Galaxy/assets/69011963/c3b9a783-bd66-4d92-8705-4e890a4c64c7" width="1440"/>

I can see you put the meme images well but when a user comes to your page first time. They have no idea which button to press. Please make this webpage user friendly.

Reply:
Hi @Rustamxon7 
Thank you for your feedback. I designed this part of the page following the design attached. Nevertheless, I really appreciate your input regarding making the app user-friendly, which is why I made a few changes that include adding the arrow icon and the borders around the emojis to make them look like cards
![image](https://github.com/MarwanDev/Meme-Galaxy/assets/46492634/ae5a1a0d-9543-4803-baff-b106c8d19b49)

Reply:
It looks great!